src/settingsui.ts

```diff
@@ -533,19 +595,144 @@ export function renderSettings() {
 				};
 				if (userscript.meta) { // render custom metadata if provided
 					const thisMeta = userscript.meta;
+					// Define low-scope variables because I can't be arsed to copy + paste the same ternary operator
+					var scriptAuthor = ('author' in thisMeta && thisMeta.author) ? `${thisMeta.author}` : false
```

Comment:
again, why var? lol

Reply:
Lazy programming? After doing some research on `var`, I know why not to use it. I'm familiar with `const` but not `let`. I'll refactor and replace.